src/rviz_visual_tools.cpp

```diff
@@ -2754,6 +2754,22 @@ void RvizVisualTools::generateEmptyPose(geometry_msgs::Pose& pose)
   pose.orientation.w = 1;
 }
 
+geometry_msgs::Pose RvizVisualTools::getIdentityPose()
+{
+  geometry_msgs::Pose pose;
+  // Position
+  pose.position.x = 0;
+  pose.position.y = 0;
+  pose.position.z = 0;
+
+  // Orientation on place
```

Comment:
**on place** is a little confusing to me. Maybe just leave as `//orientation`

Reply:
```suggestion
  // Orientation
```